qa/rpc-tests/bumpfee.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# Copyright (c) 2016 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+
+class BumpFeeTest (BitcoinTestFramework):
+
+    def setup_chain(self):
+        print("Initializing test directory "+self.options.tmpdir)
+        initialize_chain_clean(self.options.tmpdir, 4)
```

Comment:
Nit: I think you only need 3 nodes?

Also, you can use `__init__` now. (Instead of overwriting `setup_chain`):

```py
    def __init__(self):
        super().__init__()
        self.num_nodes = 3
        self.setup_clean_chain = True # not needed (default should already be true)
```